Grammar fix in readme.md (#1378)
TypeStrong/typedoc: README.md

-   `--options`<br>
    Specify a json option file that should be loaded. If not specified TypeDoc will look for 'typedoc.json' in the current directory.
-   `--json <path/to/output.json>`<br>

>     Specifies the location and file name of a json file describing the project is written to. When specified no documentation will be generated.

Reviewer: ```suggestion
    Specifies the path to write the json file describing the project. When specified, no documentation will be generated - only this json file will be.
```
